api-report/sequence.api.md

```diff
@@ -106,7 +106,13 @@ export interface IEndpointIndex<TInterval extends ISerializableInterval> extends
 // @public
 export interface IEndpointInRangeIndex<TInterval extends ISerializableInterval> extends IntervalIndex<TInterval> {
     // (undocumented)
-    findIntervalsWithEndpointInRange(start: number, end: number): any;
+    findIntervalsWithEndpointInRange(start: number | {
```

Comment:
I think the query range with `segment + offset` is more common for the SharedString users? Considering the `createEndpointInRangeIndex` is specific to SharedString right now, would it be appropriate to include this implementation? I am not sure but it may look weird to query with plain-number for SharedString..?

Reply:
I'd rather fix the signature in a way that's unified with Connnor's recent changes, so eventually this should take in `number | { segment: ISegment, offset: number } | "start" | "end"`. Dunno if enough of that work has landed in main that it makes snese yet. It's still kinda unrelated to this change though--usually it's best to keep changes semantically related.